### PR TITLE
PIMS-105 Update doc and provide method to tag images.

### PR DESCRIPTION
**Pull request by devinleighsmith**
provide pipeline assisted method of applying tags to built images.
-Update doc to explain how to check sha on deployed images.

### CODE REVIEW

File: openshift/pipelines/Jenkinsfile.cicd [hunk 1]
```diff
@@ -358,6 +358,64 @@ pipeline {
...
+    stage("Tag API") {
+      environment {
+        COMP_NAME = "api"
+      }
+      steps {
+        script {
+          openshift.withCluster() {
+            openshift.withProject() {
+              try {
+                timeout(time: 1, unit: "MINUTES") {
```
Comment (Fosol) [inline openshift/pipelines/Jenkinsfile.cicd:370]: I think this now expects us to enter a tag for all pipelines that use this Jenkinsfile.  Do we want to do this in our other environments?  I'd think for DEV we'd just want to keep using the `dev` tag.

File: openshift/templates/maintenance/caddy.bc.json [hunk 2]
```diff
@@ -0,0 +1,110 @@
+{
```
Comment (Fosol) [inline openshift/templates/maintenance/caddy.bc.json:1]: Please convert to yaml to keep it the same as our other templates and configs.

File: openshift/templates/maintenance/caddy.dc.json [hunk 3]
```diff
@@ -0,0 +1,136 @@
+{
```
Comment (Fosol) [inline openshift/templates/maintenance/caddy.dc.json:1]: Please convert to yaml to keep it the same as our other templates and configs.

File: openshift/pipelines/Jenkinsfile.cicd (hunk 1 above)
Comment (devinleighsmith) [reply openshift/pipelines/Jenkinsfile.cicd:370]: Right, let me explain exactly what this is doing.
1) If you don't enter a tag the current functionality of the pipeline is unchanged.
2) If you enter a tag, a new tag will be created in addition to any tags normally created by the pipeline. The new phases is ignored by all future phases of the pipeline, but it is essentially "saved" for historic reference.

File: openshift/templates/maintenance/caddy.bc.json (hunk 2 above)
Comment (devinleighsmith) [reply openshift/templates/maintenance/caddy.bc.json:1]: uhhh, great. These force pushes were supposed to be on a separate branch. let me clean this up.

Comment (devinleighsmith) [reply openshift/templates/maintenance/caddy.bc.json]: moved all of this out, will add in it's own PR later.

**Pull request merged**